mkosi.images/initrd/mkosi.conf.d/centos-fedora.conf

```diff
@@ -12,3 +12,9 @@ VolatilePackages=
         systemd
         systemd-libs
         systemd-udev
+        # Pull in systemd-container so that the import-generator is available
+        systemd-container
+        # Pull in systemd-networkd so that we can do network boot
+        systemd-networkd
+        # Pull in systemd-resolved so that _gateway works
+        systemd-resolved
```

Comment:
You should add these to the debian and opensuse files as well. 

- systemd-networkd is called systemd-network on opensuse and doesn't exist on debian/ubuntu
- systemd-container is named the same on debian/ubuntu/opensuse
- systemd-resolved is named the same on debian/ubuntu and part of systemd-network on opensuse